crux-core/src/crux/node.clj

```diff
@@ -89,7 +89,10 @@
   (status [this]
     (cio/with-read-lock lock
       (ensure-node-open this)
-      (status-fn)))
+      (if status-fn
+        (status-fn)
+        ;; fallback for before status-fn's set
```

Comment:
No need for comment?

Reply:
I wondered whether this was counter-intuitive? if not, happy to remove it

Reply:
have updated the comment to explain _why_ it's counter-intuitive - this one only said what it was doing - HTH?